numba/targets/arraymath.py

```diff
@@ -999,3 +1019,192 @@ def dot_3(context, builder, sig, args):
         return dot_3_vm(context, builder, sig, args)
     else:
         assert 0
+
+
+def call_xxgetrf(context, builder, a_type, a_shapes, a_data, ipiv, info):
+    """
+    Call the LAPACK gettrf function for the given arguemnt.
+
+    This function compute the LU decomposition of a matrix.
+
+    """
+    fnty = ir.FunctionType(ir.IntType(32),
+                           [ll_char,                       # kind
+                            intp_t, intp_t,                # m, n
+                            ll_void_p, intp_t,             # a, lda
+                            ll_intc_p, ll_intc_p           # ipiv, info
+                           ])
+
+    fn = builder.module.get_or_insert_function(fnty, name="numba_xxgetrf")
+
+    kind = get_blas_kind(a_type.dtype)
+    kind_val = ir.Constant(ll_char, ord(kind))
+
+    if a_type.layout == 'F':
+        m, n = a_shapes
+        lda = a_shapes[0]
+    else:
+        n, m = a_shapes
+        lda = a_shapes[1]
+
+    res = builder.call(fn, (kind_val, m, n,
+                            builder.bitcast(a_data, ll_void_p), lda,
+                            ipiv, info
+                            ))
+    check_lapack_return(context, builder, res)
+
+
+def call_xxgetri(context, builder, a_type, a_shapes, a_data, ipiv, work,
+                 lwork, info):
+    """
+    Call the LAPACK gettri function for the given arguemnt.
+
+    This function compute the invers of a matrix given its LU decomposition.
```

Comment:
compute**s** the invers**e** of a matrix from its LU decomposition as computed by `xxgetrf`.